README.md

```diff
@@ -89,13 +89,32 @@ Tests and metrics
 
 To run all required test you can run the following commands from the dev env:
 
-``` 
+```bash 
     ./bin/bootstrap_phantomjs.sh
     composer test 
 ```
 
 Every part can be run separately. Check "scripts" section of the composer.json file for the different options.
 
+Release instructions example project
+====================================
+
+The projects follow semantic versioning. To create a new release perform the following steps:
+
+1. Merge feature branch in development branch.
+
+2. Merge development branch in master branch.
```

Comment:
Most other SURFnet projects utilize a release branch to get data from develop into master. This might be overkill in this example project.

Reply:
Yea was thinking about, i will add this to the GSSP bundle